docs/source/install.rst

```diff
@@ -0,0 +1,225 @@
+
+#. Pick the preferred commit from `commit list <https://github.com/luxonis/depthai-python/commits/gen2_develop>`__
+
+#. Upgrade pip
+
+  .. code-block:: bash
+
+    python3 -m pip install -U pip
+
+#. Install DepthAI Gen2 API, adding commit SHA at the end of a package name, in form of :code:`depthai==0.0.2.1+<sha>`
+
+  .. code-block:: bash
+
+    python3 -m pip install --extra-index-url https://artifacts.luxonis.com/artifactory/luxonis-python-snapshot-local/ depthai==0.0.2.1+6ec3f3181b4e46fa6a9f9b20a5b4a3dac5e876b4
+
+For other installation options, see :ref:`other installation options <Other installation methods>`.
+
+Test installation
+#################
+
+We have `a set of examples <https://github.com/luxonis/depthai-python/tree/gen2_develop/examples>`__ that should help you verify if your setup was correct.
+
+First, clone the `depthai-python <https://github.com/luxonis/depthai-python/tree/gen2_develop>`__ repository and change directory into this repo:
+
+.. code-block:: bash
+
+  git clone https://github.com/luxonis/depthai-python.git --branch gen2_develop
+  cd depthai-python
+
+Now, run the :code:`01_rgb_preview.py` script from within :code`examples` directory to make sure everything is working:
+
+.. code-block:: bash
+
+  python3 examples/01_rgb_preview.py
+
+If all goes well a small window video display will appear with color camera preview
+
+
+.. warning::
+
+  Some of the examples needs additional setup to be ran. Please be sure to follow `setup instructions in README.md <https://github.com/luxonis/depthai-python/tree/gen2_develop/examples#depthai-python-examples>`__
+  to run different examples
+
+Other installation methods
+##########################
+
+To get the latest features from our source code, you can go ahead and compile depthai package manually.
+
+Dependencies to build from source
+*********************************
+
+- CMake > 3.2.0
+- Generation tool (Ninja, make, ...)
+- C/C++ compiler
+- libusb1 development package
+
+.. _raspbian:
+
+Ubuntu, Raspberry Pi OS, ... (Debian based systems)
+---------------------------------------------------
+
+On Debian based systems (Raspberry Pi OS, Ubuntu, ...) these can be acquired by running:
+
+.. code-block:: bash
+
+  sudo apt-get -y install cmake libusb-1.0-0-dev build-essential
+
+macOS (Mac OS X)
+----------------
+
+Assuming a stock Mac OS X install, `depthai-python <https://github.com/luxonis/depthai-python/tree/gen2_develop>`__ library needs following dependencies
+
+- Homebrew (If it's not installed already)
+
+  .. code-block:: bash
+
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+
+- Python, libusb, CMake, wget
+
+  .. code-block:: bash
+
+      brew install coreutils python3 cmake libusb wget
+
+And now you're ready to clone the `depthai-python <https://github.com/luxonis/depthai-python/tree/gen2_develop>`__ from Github and build it for Mac OS X.
```

Comment:
Take branch from git rather than hardcoding?

Reply:
I want to reference the gen2_develop as long as we don't make a release or move completely to gen2 (and then use main / develop branches). Making a link straight to gen2_develop will hopefully create less confusion. Will add a note that the gen2_develop branch is targeted